distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,120 @@
+priority "grant";
+// todo symlinks
+
+deny {
+    // Read, write, or execute any file
+    permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
+
+    // Deny deletion of the security policy file through any Java code
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}default.policy", "delete";
+
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
```

Comment:
migration export/import needs read access to:

javax.net.ssl.keyStore
javax.net.ssl.trustStore

Reply:
Let me know which bundle(s) need those permissions and I'll grant them.

Reply:
It's a place holder for now. I will compile all the requirements for export/import separatly

Reply:
Can you add the following permissions for export/import?

//
// DDF Migration Framework and Config Admin Migratable
//
grant
  codeBase "file:${ddf.home.policy}system/ddf/platform/platform-migratable/-" {
    // some permissions might be repeated here to ensure that they are granted to the migration
    // framework if we ever tighten the permissions above

    permission java.io.FilePermission "<<ALL FILES>>", "read, readlink";
    permission java.io.FilePermission "${ddf.home}", "read, write";

    permission java.util.PropertyPermission "*", "read";
    permission java.util.PropertyPermission "ddf.home", "read";
    permission java.util.PropertyPermission "karaf.restart.jvm", "write";
};

//
// DDF Platform Migratable
//
grant
  codeBase "file:${ddf.home.policy}system/ddf/platform/platform-migratable/-" {
    // some permissions might be repeated here to ensure that they are granted to the migration
    // framework if we ever tighten the permissions above

    permission java.io.FilePermission "${ddf.home}${/}etc${/}ws-security", "read";
    permission java.io.FilePermission "${ddf.home}${/}etc${/}ws-security${/}-", "read, delete";

    permission java.util.PropertyPermission "javax.net.ssl.keyStore", "read";
    permission java.util.PropertyPermission "javax.net.ssl.trustStore", "read";
};

//
// DDF Security Migratable
//
grant
  codeBase "file:${ddf.home.policy}system/ddf/security/migration/security-migratable/-" {
    // some permissions might be repeated here to ensure that they are granted to the migration
    // framework if we ever tighten the permissions above

    permission java.io.FilePermission "${ddf.home}${/}etc${/}pdp", "read";
    permission java.io.FilePermission "${ddf.home}${/}etc${/}pdp{/}-", "read, delete";
};

Reply:
Will do, @paouelle